### DISCUSSION TITLE
请教一下，这种斜体被截断的情况，有办法解决不

### DISCUSSION THREAD

**Discussion by huauh** [nshen/learn-neovim-lua#143] [Q&A]
![斜体字符被截断](https://user-images.githubusercontent.com/21192376/202178417-8aa9f83d-321f-4521-97fb-39e13a0e8e44.png)

Windows 终端，版本: 1.15.2875.0
版本	Windows 11 专业版
版本	22H2
安装日期	‎2022-‎05-‎20
操作系统版本	22623.891
体验	Windows Feature Experience Pack 1000.22637.1000.0

**Comment by nshen** [comment]
好像是因为需要等宽字体，名字带 mono 的，你试试可以不

**Reply by huauh** [reply]
- 试了下带mono的字体，还是不行
 ![字体](https://user-images.githubusercontent.com/21192376/202185399-a0d7f004-ccc9-42d0-b3e3-5472f149fadd.png)
- 我注意到您的教程里，也是被截断的
![教程截图](https://user-images.githubusercontent.com/21192376/202185983-0c35755d-164a-4bb8-8262-2c03998b633f.png)

**Comment by nshen** [comment]
是的我之前注意过这个问题，但我有点记不清当时是怎么解决的了，后来换了mac后也没遇到过这个问题，不知道其他用WSL的大佬们有遇到这个问题吗？

**Comment by lifei** [comment]
解决了吗？
